test/index.test.js

```diff
@@ -27,9 +27,30 @@ describe('auto-cookie', () => {
   })
 
   it('should not set cookie around 0.0.0.0', done => {
-    browser.url('http://0.0.0.0:8000').getCookie(name).then(cookie => {
-      assert(!cookie)
-      done()
-    })
+    browser
+      .url('http://0.0.0.0:8000')
+      .getCookie(name)
+      .then(cookie => {
+        assert(!cookie)
+        done()
+      })
+  })
+})
+
+describe('path-cookie', () => {
```

Comment:
please 1 file single describe, because `.only` options is effective range over callbacks